cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
+  },
+  video: true,
+  viewportHeight: 1920,
```

Comment:
The 'viewportHeight' and 'viewportWidth' values seem to be swapped. Generally, width is larger than height for screen resolutions. You might want to set 'viewportWidth' to 1920 and 'viewportHeight' to 1080.